### PR TITLE
[UI][Theme] Add flag to make "About CDAP" modal configurable.

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Adds a feature flag to show/hide About CDAP modal in UI
- Updates theme files to have the newly added flag with default value.
- Fixes the Hub button to have proper margins in different sections of the UI (Admin -> Configuration for instance)

Build: https://builds.cask.co/browse/CDAP-URUT143

### CODE REVIEW

File: cdap-ui/app/cdap/components/Header/ProductDropdown/index.js [hunk 1]
```diff
@@ -148,14 +149,16 @@ export default class ProductDropdown extends Component {
...
           <CustomDropdownMenu right>
-            <DropdownItem
-              tag="li"
-              onClick={this.toggleAboutPage}
-            >
-              <a>{T.translate('features.Navbar.ProductDropdown.aboutLabel', {
-                productName: Theme.productName,
-              })}</a>
-            </DropdownItem>
+            <If condition={Theme.showAboutProductModal}>
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Header/ProductDropdown/index.js]: nit: we are being explicit in other places `Theme.showAboutProductModal === true`

**Review by elfenheart — COMMENTED** [review]
one minor comment

File: cdap-ui/server/config/themes/light.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
     "navbar-color": "#3b78e7"
   },
   "features": {
+    "about-product-modal": true,
```
Comment (nitinmotgi) [inline cdap-ui/server/config/themes/light.json]: Can we just use about-product please

**Review by nitinmotgi — CHANGES_REQUESTED** [review]
Minor change

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Comment by ajainarayanan** [comment]
Latest green build: https://builds.cask.co/browse/CDAP-URUT143-4

**Comment by nitinmotgi** [comment]
LGTM 👍

**Review by nitinmotgi — APPROVED** [review]

**Pull request merged**